composer.json

```diff
@@ -21,7 +21,9 @@
     },
     "require": {
         "php": ">=5.4.0",
-        "nikic/php-parser": "^2.1|^3.0|^4.2"
+        "nikic/php-parser": "^2.1|^3.0|^4.2",
+        "phing/phing": "2.*",
+        "phpunit/phpunit": "4.8.16"		
```

Comment:
@biechao Remove the change. All users don't want to use Phing, and PHPUnit 4.8.16.

Reply:
The report needs Phing as dependence, it must be installed via composer.
For PHPUnit 4.8.16, it should be run as global as my previous comment.

Reply:
@biechao You should remove the changes in `composer.json`. Because ci-phpunit-test does not need Phing.
It is optional.

But your project needs Phing. So add instructions to install it via Composer on the README.